README.md

```diff
@@ -140,6 +140,19 @@ $ clang \
 -o example example.c
 ```
 
+The builds of the toolchain come packaged with two config files, Omax.cfg and OmaxLTO.cfg.
+When used, these config files enable several build optimisation flags. OmaxLTO.cfg enables LTO
```

Comment:
Would be nice to explain:
...optimization flags to achieve highest performance on typical embedded benchmarks.

Reply:
Done!